python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -596,7 +606,9 @@ def process_node_data(process_confs, arr_merger, remap_id,
                                     ext_mem_workspace_type)
         type_node_id_map = [None] * len(return_dict)
         type_node_data = {}
-        for i, (node_ids, data) in return_dict.items():
+        for i, (node_ids, data, feat_dim_dict) in return_dict.items():
+            if process_conf and ("features" in process_conf):
+                update_feat_transformation_conf(process_conf["features"], feat_dim_dict)
```

Comment:
Can we add a comment to explain this change?

Reply:
We can add "Write feature dim to the gconstruct config"

Reply:
Added.